guides/common/modules/proc_upgrading-project-or-proxy-in-place-using-leapp.adoc

```diff
@@ -0,0 +1,230 @@
+[id="upgrading-{project-context}-or-proxy-in-place-using-leapp_{context}"]
+= Upgrading {Project} or {SmartProxy} to {EL} 9 in-place by using Leapp
+
+When upgrading your {Project} or {SmartProxy} installation from {EL} 8 to {EL} 9, use the Leapp tool to help detect and resolve issues that could prevent you from upgrading successfully.
```

Comment:
- So is Leapp just for a pre-upgrade check or can it be used for the whole upgrade? The conclusion of this sentence doesn't match the module heading. Can you please try to be more precise about the use case?
- Will this also upgrade the Satellite or only the OS?
- Why should the user go with this procedure and not the other ones? Perhaps this is something that should also be addressed in the _Upgrade Paths_?

Reply:
@Lennonka so Leapp will check for issues as well as upgrade the OS (not the Satellite). I have added this information now.